Automattic/wp-calypso
# Help Center: fix audio playing in embedded HC

#### Proposed Changes

* This adds the pling noise file to Happychat app in `apps/happychat` and copies it as part of the build process.
* It makes the path of the file configurable.
* It imports the audio middleware for embedded Happychat.

#### Testing Instructions

1. Open the Help Center.
2. Login to https://hud-staging.happychat.io/ to accept your own session. 
3. Start a chat session. 
4. Send a message from the Help Center first. 
5. Send a message to yourself from the HUD. 
6. You should hear a noise.
